libpdbg/device.c

```diff
@@ -48,11 +48,119 @@ static const char *take_name(const char *name)
 	return name;
 }
 
+static const char *get_unitname(const struct pdbg_target *node)
+{
+	const char *c = strchr(node->dn_name, '@');
+
+	if (!c)
+		return NULL;
+
+	return c + 1;
+}
+
+/* The moment any number is found, in a string:
+   separate the string into number and string without numbers.
+   Return the number found, update the string passed to chop the number from it
+   Find only the number at the end of the string*/
+unsigned int strip_num_at_the_end(char *str) 
```

Comment:
If this function is only called by the functions in device.c then it is better to make it static.